spec/rails_helper.rb

```diff
@@ -37,7 +37,8 @@
   config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
   config.hook_into :webmock
   config.default_cassette_options = {
-    match_requests_on: [:uri, :body, :method]
+    match_requests_on: [:uri, :body, :method],
+    record: :new_episodes
```

Comment:
This setting allow replaying previously recorded entries, and if they don't exist, to record a new one to be replied. I added this setting because due to the mounted volumes issue I have with compose, I was getting a lot of requests that VCR didn't know how to handle on the existing cassettes.